spec/openapi.yaml

```diff
@@ -125,6 +125,148 @@ paths:
   /translation-quality-api/v2/accounts/{accountUid}/check-types/GLOSSARY_COMPLIANCE/groups/{groupUid}/settings:
     $ref: './spec/translation_quality/checks/glossary_compliance.yaml#/components/schemas/check_types_GLOSSARY_COMPLIANCE_group_settings'
 
+#  /translation-quality-api/v2/accounts/{accountUid}/check-types/{checkTypeCode}/test:
+#    get:
+#      description: >-
+#        tolik Get checktype settings for certain account.
```

Comment:
"tolik"?

Reply:
This is commented in rows. It will be cleaned before merging to master.